_extras/motivation.md

```diff
@@ -2,5 +2,46 @@
 title: Motivation
 ---
 
+# Cross Stitching is like computer programming
+
+Computer programming or coding requires analytical skills, which allows us to think of a solution to a problem, design a plan comprising of multiple steps and accomplish them by writing multiple lines of code. 
+Each of these lines have been intentionally written to play a vital role in a working software.
+
+Cross stitching is quite like programming where you create a beautiful pattern by "writing" in rows. 
+If there’s a bug or mis-stitching in a row, you have to fix that to create your pattern as you intented to look like.
+
+Keeping that in mind, we invite you to explore cross-stitching as a hobby for programmers. 
+This lesson is designed to train a novice learner to learn how to cross stitch and use it as a hobby to mindfully enjoy their breaks from work.
+
+# Why cross stitch as a programmer?
+
+On a daily basis, we use smartphone and computers for everything we do. 
+Starting from getting a fast access to news and latest information in the morning, to learning a new programming language or other skills to getting our job done at work, to ordering things that makes our day-to-day life possible.
+Needless to say, such excessive use of technology leads to information overload, and often results into burnout and other mental health issues including anxiety, and depression [1].
+
+<SARAH HELP ME>
```

Comment:
```suggestion
## Sarah's Impact Story

I was taught to cross stitch in my early teens by my aunt.
It was also the same time she got me into heavy metal music and took to me to my first rock music gigs.
She is still by far the coolest member of my family!

I stuck with cross stitching as a hobby for a number of reasons, but most of all I do it because it keeps my hands busy.
In an evening, I would often find myself mindlessly scrolling through social media when I was supposed to be paying attention to a film or a conversation with my friend.
Cross stitching keeps me off my phone when I don't need to be, but still leaves me with enough "brain space" to carry on a conversation or watch something.
(My new favourite thing to stitch along to is audiobooks!)

Another reason I have continued to cross stitch is that it forces me to slow down.
If I rush stitching, I'm bound to make a mistake.
So instead I take my time to make sure I have the right coloured thread and I've counted the correct number of spaces. _Count twice, stitch once!_
```